### DISCUSSION TITLE
RabbitMQ version in kubernetes

### DISCUSSION THREAD

**Discussion by PHGamer** [rabbitmq/rabbitmq-server#9686] [Questions] [answered]
How does one specify the version of rabbit using the cluster operator?

I assume its in the defintion.yaml but not sure what the key would be?

**Comment by michaelklishin** [comment, marked as answer]
By [specifying the image and the tag that should be used](https://rabbitmq.com/kubernetes/operator/using-operator.html#images).

**Reply by michaelklishin** [reply]
In case it is not obvious, [this is the most widely used image](https://hub.docker.com/_/rabbitmq/) but there are others, such as the [Bitnami one](https://hub.docker.com/r/bitnami/rabbitmq), the [commercial edition](https://rabbitmq.com/kubernetes/tanzu/installation.html) image, and so on.

If you use open source RabbitMQ, the community image is a good place to start.
